assets/js/admin-pull.js

```diff
@@ -82,12 +94,14 @@ if ( chooseConnection && choosePostType && form ) {
  * @return {string} Distribution URL.
  */
 const getURL = () => {
-	const postType =
-		choosePostType.options[ choosePostType.selectedIndex ].value;
-	const baseURL =
+	const postType = escapeURLComponent(
+		choosePostType.options[ choosePostType.selectedIndex ].value
+	);
+	const baseURL = escapeURLComponent(
```

Comment:
This changes the URL from absolute to relative so the search form ends up redirecting to something like the followiong and throwing a 404.:

```
http://ms-distributor.local/wp-admin/http%3A%2F%2Fms-distributor.local%2Fwp-admin%2Fadmin.php%3Fpage%3Dpull%26connection_type%3Dinternal%26connection_id%3D2&pull_post_type=post&status=new&s=loggign
```